docs/src/components/Guidelines/index.tsx

```diff
@@ -1,161 +1,146 @@
 import styled, { css } from "styled-components";
 import { CheckCircle, CloseCircle } from "@kiwicom/orbit-components/icons";
-import { Stack, Text, Heading } from "@kiwicom/orbit-components";
-import { imageWrapperClass } from "gatsby-remark-images/constants";
+import {
+  Stack,
+  Text,
+  Heading,
+  Grid,
+  useMediaQuery,
+  mediaQueries as mq,
+} from "@kiwicom/orbit-components";
+import useTheme from "@kiwicom/orbit-components/lib/hooks/useTheme";
 
 import HeadingWithLink from "../HeadingWithLink";
 import { slugify } from "../../utils/common";
+import { extractContent, resolveBorders, resolveColumns } from "./helpers";
 
 export interface GuidelineType {
   type: "do" | "dont";
 }
-
+export interface Content {
+  images: React.ReactNode[];
+  content: React.ReactNode;
+}
 interface GuidelineProps extends GuidelineType {
   title: string;
   children: React.ReactNode;
 }
 
-const StyledComponent = styled.div`
-  ${({ theme }) => css`
-    background-color: ${theme.orbit.paletteCloudLight};
-    border-radius: ${theme.orbit.spaceMedium};
-    padding: ${theme.orbit.spaceMedium} 0;
-  `}
-`;
-
-interface WrapperProps extends GuidelineType {
-  border: boolean;
+interface GuidelineComponent extends GuidelineType {
+  id: string;
+  coloredBorder: boolean;
 }
 
-const Wrapper = styled.div<WrapperProps>`
-  ${({ border, theme, type }) => css`
-    ${border &&
-    `border-left: 4px solid ${
-      type === "do" ? theme.orbit.paletteGreenNormal : theme.orbit.paletteRedNormal
-    }`};
-    padding: ${theme.orbit.spaceXSmall} ${theme.orbit.spaceLarge} ${theme.orbit.spaceXSmall}
-      ${theme.orbit.spaceMedium};
-    display: flex;
-  `}
-`;
+interface ImageContainerProps extends GuidelineType {
+  middleAlign?: boolean;
+}
 
-const ContentContainer = styled.div`
+const StyledComponent = styled.div<GuidelineComponent>`
   ${({ theme }) => css`
-    padding-left: ${theme.orbit.spaceXSmall};
+    background: ${theme.orbit.paletteCloudLight};
+    border-radius: ${theme.orbit.borderRadiusLarge};
+    padding: ${theme.orbit.spaceMedium} 0;
+    ${resolveBorders};
   `}
 `;
 
-interface ImageContainerProps {
-  noLeftPadding?: boolean;
-  middleAlign?: boolean;
-}
-
-const ImageContainer = styled.div<ImageContainerProps>`
-  ${({ noLeftPadding, theme }) => css`
-    padding: ${noLeftPadding ? theme.orbit.spaceMedium : theme.orbit.spaceLarge};
-    ${noLeftPadding && "padding-left: 0;"}
-    width: 100%;
-    max-width: 360px;
-    background-color: ${theme.orbit.paletteWhite};
-    border-radius: ${theme.orbit.spaceMedium};
+const StyledWrapper = styled.div<GuidelineType>`
+  ${({ theme }) => css`
+    display: flex;
+    padding: ${theme.orbit.spaceXSmall} ${theme.orbit.spaceLarge} ${theme.orbit.spaceXSmall}
+      ${theme.orbit.spaceMedium};
   `}
 `;
 
-const ImageBorder = styled.div<GuidelineType>`
+const StyledImageContainer = styled.div<ImageContainerProps>`
   ${({ theme, type }) => css`
-    border-left: 4px solid
+    padding: 10%;
```

Comment:
This padding is a little much vertically on smaller resolutions:

![Screen Shot 2021-09-15 at 18 17 20](https://user-images.githubusercontent.com/471278/133470871-b92d0524-57c7-4493-8646-5a11afda9190.png)

I think it should stay `spaceMedium` and you could add a maximum width of the image if you can, and center it.